components/Flipper/Flipper.tsx

```diff
@@ -44,9 +44,15 @@ interface Props {
 }
 
 function Flipper(props: Props) {
-    let [flips, setFlips] = useState<FlipAuction[]>(props.flips || [])
-    let [isLoggedIn, setIsLoggedIn] = useState(false)
     let [flipperFilter, setFlipperFilter] = useState<FlipperFilter>(getSettingsObject<FlipperFilter>(FLIPPER_FILTER_KEY, {}))
+    let [flips, setFlips] = useState<FlipAuction[]>(
```

Comment:
What about new flips getting sold?
The statehook is only created once when the site loads, isn't it?